{'project_id': '1211', 'year': '1989', 'x': 961140, 'y': 1328220, 'plot_id': '456', 'strata': '41'}
Pandas(Index=0, project_id='1211', plot_id='456', x=961140, y=1328220)
called process_plot
Pandas(Index=0, project_id='1211', plot_id='456', x=961140, y=1328220)
456
Directory '/efs/timesync/1989/audit/' already exists.
Data has been pickled and saved to '/efs/timesync/1989/audit/1989_0456_1211.p'.
.....................Failed qa: 1989-03-30T15:41:32.335091Z
.........................................................................................................Failed qa: 1989-09-22T15:38:29.4170000Z
.......Failed qa: 1989-10-24T15:37:21.296056Z
.....................Data has been pickled and saved to '/efs/timesync/1989/audit/1989_0456_1211_qa_rejects.p'.
Success tracking goes here
